### PR TITLE
Expose service-name and service-roles in the CloudLibrary json binding

### PR DESCRIPTION
**Pull request by etanshaul**
Follow up to #483 and #485 

Exposes the new metadata in the json binding

### CODE REVIEW

File: src/main/java/com/google/cloud/tools/libraries/json/CloudLibrary.java [hunk 1]
```diff
@@ -45,6 +47,18 @@ public String getId() {
     return id;
   }
 
+  /** Returns the serviceName associated with this library. */
```
Comment (elharo) [inline src/main/java/com/google/cloud/tools/libraries/json/CloudLibrary.java]: service name?

File: src/main/java/com/google/cloud/tools/libraries/json/CloudLibrary.java [hunk 2]
```diff
@@ -45,6 +47,18 @@ public String getId() {
...
+  @Nullable
+  public String getServiceName() {
+    return serviceName;
+  }
+
+  /** Returns the serviceRoles associated with this library. */
```
Comment (elharo) [inline src/main/java/com/google/cloud/tools/libraries/json/CloudLibrary.java]: service roles?

File: src/main/java/com/google/cloud/tools/libraries/json/CloudLibrary.java (hunk 1 above)
Comment (etanshaul) [reply src/main/java/com/google/cloud/tools/libraries/json/CloudLibrary.java]: fixed

File: src/main/java/com/google/cloud/tools/libraries/json/CloudLibrary.java (hunk 2 above)
Comment (etanshaul) [reply src/main/java/com/google/cloud/tools/libraries/json/CloudLibrary.java]: fixed

**Comment by etanshaul** [comment]
@elharo good with this now?

**Comment by etanshaul** [comment]
@elharo this is simple enough so I'll go ahead and merge

**Pull request merged**